

2017-02-18 13:56:58+0100 [-] ....Received FRAME message of length 13030.
2017-02-18 13:56:58+0100 [-] ..
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] Error getting result from Torch subprocess.
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] Line read: 6453476,0.21842414140701,-0.0030650566332042,-0.021574730053544,0.10138932615519,0.049214031547308,-4,0.11566124856472,-0.070597961544991,-0.12099632620811,0.068237975239754,0.15849792957306,0.064384058117867,0.0042726513929665,-0.093476220965385,-0.18826767802238,-0.16992717981339,0.12724573910236,0.039518840610981,-0.11023943126202,-0.096253968775272,0.043253879994154,-0.052826061844826,0.055508762598038,-0.093464858829975,0.066015899181366,-0.10706739872694,0.050460.080931425094604,-0.046889387071133,-0.08226466178894,-0.039233885705471,0.042012896388769,-0.010918387211859,0.051816072314978,0.12199904769659,0.16448578238487,-0.050497461110353,0.036785051226616,0.028575830161572,-0.071268908679485,0.048551842570305,-0.052292101085186,-0.031067576259375,-0.11513634026051,0.067532360553741,-0.086805194616318,0.15087160468102,0.1200315207243,-0.033057175576687,0.21622218191624,-0.040588688105345,-0.12528179585934,-0.041429437696934,-0.07777101546526,0.035785429179668,0.070027880370617,0.0013615592615679,-0.18641173839569,-0.089445300400257,0.01165315322578,0.09154986590147,-0.12450985610485,0.038773335516453,-0.079193435609341,0.086982600390911,0.017942614853382,0.11134084314108,-0.04641579836607,-0.11473615467548
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] Exception:
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] invalid literal for float(): 0.050460.080931425094604
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] ============
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [-] stdout: 
2017-02-18 13:56:58+0100 [-] 
2017-02-18 13:56:58+0100 [OpenFaceServerProtocol,0,127.0.0.1] Unhandled Error
	Traceback (most recent call last):
	  File "/usr/local/lib/python2.7/dist-packages/twisted/python/log.py", line 101, in callWithLogger
	    return callWithContext({"system": lp}, func, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/twisted/python/log.py", line 84, in callWithContext
	    return context.call({ILogContext: newCtx}, func, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 118, in callWithContext
	    return self.currentContext().callWithContext(ctx, func, *args, **kw)
	  File "/usr/local/lib/python2.7/dist-packages/twisted/python/context.py", line 81, in callWithContext
	    return func(*args,**kw)
	--- <exception caught here> ---
	  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/posixbase.py", line 611, in _doReadOrWrite
	    why = selectable.doRead()
	  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/tcp.py", line 209, in doRead
	    return self._dataReceived(data)
	  File "/usr/local/lib/python2.7/dist-packages/twisted/internet/tcp.py", line 215, in _dataReceived
	    rval = self.protocol.dataReceived(data)
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/twisted/websocket.py", line 101, in dataReceived
	    self._dataReceived(data)
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 1299, in _dataReceived
	    self.consumeData()
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 1314, in consumeData
	    while self.processData() and self.state != WebSocketProtocol.STATE_CLOSED:
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 1466, in processData
	    return self.processDataHybi()
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 1777, in processDataHybi
	    fr = self.onFrameEnd()
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 1904, in onFrameEnd
	    self._onMessageEnd()
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/twisted/websocket.py", line 129, in _onMessageEnd
	    self.onMessageEnd()
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/websocket/protocol.py", line 731, in onMessageEnd
	    self._onMessage(payload, self.message_is_binary)
	  File "/usr/local/lib/python2.7/dist-packages/autobahn/twisted/websocket.py", line 132, in _onMessage
	    self.onMessage(payload, isBinary)
	  File "./demos/web/new-server.py", line 199, in onMessage
	    self.processFrame(msg['dataURL'], msg['identity'])
	  File "./demos/web/new-server.py", line 399, in processFrame
	    rep = net.forward(alignedFace).reshape(1, -1)
	  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 204, in forward
	    rep = self.forwardPath(t)
	  File "/usr/local/lib/python2.7/dist-packages/openface/torch_neural_net.py", line 171, in forwardPath
	    stdout, stderr = self.p.communicate()
	  File "/usr/lib/python2.7/subprocess.py", line 800, in communicate
	    return self._communicate(input)
	  File "/usr/lib/python2.7/subprocess.py", line 1412, in _communicate
	    self.stdin.flush()
	exceptions.ValueError: I/O operation on closed file
	
2017-02-18 13:56:58+0100 [-] WebSocket connection closed: connection was closed uncleanly (peer dropped the TCP connection without previous WebSocket closing handshake)


